K2Bridge.Tests.UnitTests/Visitors/DateRangeVisitorTests.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+namespace UnitTests.K2Bridge.Visitors
+{
+    using System.Collections.Generic;
+    using global::K2Bridge.Models.Request.Aggregations;
+    using global::K2Bridge.Tests.UnitTests.Visitors;
+    using global::K2Bridge.Visitors;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class DateRangeVisitorTests
+    {
+        [TestCase("2018-02-01", "2018-02-02", "2018-02-02", "2018-02-3", ExpectedResult = "_data | extend ['key'] = pack_array(strcat(make_datetime('2018-02-01'), '%', make_datetime('2018-02-02')),strcat(make_datetime('2018-02-02'), '%', make_datetime('2018-02-3'))), ['_range_value'] = pack_array(['dayOfWeek'] >= make_datetime('2018-02-01') and ['dayOfWeek'] < make_datetime('2018-02-02'),['dayOfWeek'] >= make_datetime('2018-02-02') and ['dayOfWeek'] < make_datetime('2018-02-3')) | mv-expand ['key'] to typeof(string), ['_range_value'] | where ['_range_value'] == true | summarize wibble by ['key'] | order by ['key'] asc")]
```

Comment:
dayOfWeek is a bit confusing since its a value between 1 to 7 (usually).
maybe change to time/date or something (but not to a keyword)

Reply:
Done